### PR TITLE
Update BoringSSL from Chromium 70.0.3538.110

### PR DESCRIPTION
**Pull request by dahlstrom-g**
b/309016038

### CODE REVIEW

**Comment by andrewsavage1** [comment]
Please create a separate bug for this instead of using the base/net update bug. Feel free to reuse/retitle b/309016038 if you want.

**Comment by dahlstrom-g** [comment]
> Please create a separate bug for this instead of using the base/net update bug. Feel free to reuse/retitle b/309016038 if you want.

done

File: third_party/boringssl/METADATA
Comment (andrewsavage1) [inline third_party/boringssl/METADATA:1]: Should third_party/boringssl/src/METADATA be deleted?

Comment (dahlstrom-g) [reply third_party/boringssl/METADATA:1]: No, it still shows the provenance of that directory.

Comment (andrewsavage1) [reply third_party/boringssl/METADATA:1]: So this PR updated //third_party/boringssl excluding //third_party/boringssl/src, and the latter still has the upstream of the separate boringssl repository? Is our long term solution to have two separate upstreams or is this temporary?

Comment (dahlstrom-g) [reply third_party/boringssl/METADATA:1]: Those two trees come from different histories in different repositories
so while that's true I think the record should reflect it.

**Comment by andrewsavage1** [comment]
As long as this also passes internal builds it seems fine

**Comment by dahlstrom-g** [comment]
> As long as this also passes internal builds it seems fine

I checked that yesterday and it will but it needs approval.

**Comment by andrewsavage1** [comment]
> > As long as this also passes internal builds it seems fine
> 
> I checked that yesterday and it will but it needs approval.

All the tests look like they failed on PS11

**Comment by dahlstrom-g** [comment]
> All the tests look like they failed on PS11

It took me a while to decide that wasn't some PlayStation. 😀
I guess you mean patch set 11 of the Gerrit change—
never mind that; I didn't intend for those to run there.
It's on submitters to make sure sure builds succeed
but reviewers are not generally responsible for that.

**Review by andrewsavage1 — APPROVED** [review]

File: third_party/boringssl/BUILD.gn [hunk 1]
```diff
@@ -1,47 +1,49 @@
...
-  configs = [ ":external_config" ]
+  visibility = [ ":*" ]  # Only targets in this file can depend on this.
   defines = [
-    "PURIFY",
-    "TERMIO",
-    "_REENTRANT",
+    "BORINGSSL_ALLOW_CXX_RUNTIME",
+    "BORINGSSL_IMPLEMENTATION",
+    "BORINGSSL_NO_STATIC_INITIALIZER",
+    "OPENSSL_SMALL",
```
Comment (kaidokert) [inline third_party/boringssl/BUILD.gn:35]: This caused a significant regression - see b/360241262

**Pull request merged**